src/frontend/types/libs/m3u8-parser/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'm3u8-parser';
```

Comment:
commit message
This tag...
Fortunately**,**
Once removed, we stop fetching **, or .**

Reply:
in fact it's `Once removed, we stop fetching and the video is updated in the store`. Thanks for the typo.